src/eval/eval.ipynb

```
@@ -181,6 +244,214 @@
+    "                            {\n",
+    "                                \"tool\": exp_tool_name,\n",
+    "                                \"parameter\": key,\n",
+    "                                \"expected\": expected_value,\n",
+    "                                \"actual\": actual_value,\n",
+    "                            }\n",
+    "                        )\n",
+    "\n",
+    "                # Check for unexpected parameters\n",
+    "                for key in act_params:\n",
+    "                    if key not in exp_params:\n",
+    "                        mismatches.append(\n",
```

Ruff (0.14.11)

Unused function argument: `inputs`

(ARG001)